modules/shutup/src/main/model.scala

```diff
@@ -50,9 +53,17 @@ enum TextType(val key: String, val rotation: Int, val name: String):
   case PublicForumMessage extends TextType("puf", 20, "Public forum message")
   case TeamForumMessage   extends TextType("tef", 20, "Team forum message")
   case PrivateMessage     extends TextType("prm", 20, "Private message")
+  case UblogPost          extends TextType("ubp", 20, "User blog post")
```

Comment:
Tbh I don't think having the distinction between team/public and blog post is necessary, and would rather merge all of them into a `LongTexts` case

Reply:
The idea is that team comms are less sensitive than global public ones, and could be moderated differently.